### DISCUSSION TITLE
How we can take backup before updating the import-map

### DISCUSSION THREAD

**Discussion by ajasingh** [single-spa/import-map-deployer#104] [Q&A]
HI @joeldenning  is there any possibility that we can take the backup of the import-map file first before updating it so avoid smooth rollback of the files

**Comment by internettrans** [comment]
Generally the way I've seen rollbacks done is via a CI/CD tool for each microfrontend - in most CI tools you can redeploy an old build in order to rollback.

Does that approach not work for your situation? I'm open to things being added to import-map-deployer that make this easier, if the CI tool approach isn't feasible.
